Runtimes/Resync.cmake

```diff
@@ -97,6 +96,20 @@ foreach(library ${CoreLibs})
   copy_library_sources(${library} "public" "Core")
 endforeach()
 
+# copying Core/core separately, so we can amend the case of the target folders
+# on case insensitive file systems
+if(IS_READABLE "${CMAKE_CURRENT_LIST_DIR}/Core/core")
```

Comment:
This will always get hit on a case-insensitive filesystem regardless of whether it's `Core/core` or `Core/Core` since it's readable by both names. `add_subdirectory(Core)` will still work if someone has already copied things over with the lowercase `core`. I don't think we need to do this swap.

Reply:
On further inspection, the behavior I'm seeing is specific to some vendor workflows that enforce [case sensitivity at process level](https://worthdoingbadly.com/casesensitive-iossim/), not related to CMake as I wrongly surmised.